### DISCUSSION TITLE
DataGrid Blank

### DISCUSSION THREAD

**Discussion by Hub-moon** [AvaloniaUI/Avalonia#13192] [Q&A] [answered]
DataGrid Blank

**Comment by timunie** [comment, marked as answer]
Have you done the Style include?

**Reply by Hub-moon** [reply]
no

**Reply by Hub-moon** [reply]
Follow the example of the document

**Reply by timunie** [reply]
https://docs.avaloniaui.net/docs/next/reference/controls/datagrid/#include-data-grid-styles-

**Reply by Hub-moon** [reply]
Thank you very much. It's the lack of plugins
